tests/unit/models/factories/test_analysis_factory.py

```diff
@@ -34,15 +81,11 @@ def test_model_has_ccc_id(self, analysis_model):
         assert hasattr(analysis_model, 'cell_count_calibration_id')
 
     def test_model_can_serialize(self, analysis_model):
-        serial = AnalysisModelFactory.get_serializer().serialize(analysis_model)
-        assert len(serial) == 2
+        serial = dumps(analysis_model)
+        assert len(json.loads(serial)) == 2
 
     def test_model_can_deserialize(self, analysis_serialized_object):
-        result = AnalysisModelFactory.get_serializer().load_serialized_object(
-            analysis_serialized_object,
-        )
-        assert len(result) == 1
-        model = result[0]
+        model: AnalysisModel = loads(analysis_serialized_object)
```

Comment:
As we've discussed previously, this test seemed to actually only test things that were set to defaults (row 90+). If this is still the case, then adding something like email (and perhaps not check the defaults at all?) would be a welcome addition, while we remember it. This may apply to more tests (though I cannot think of any), but I know it was the case for this one in particular.

Reply:
fixed